service.subtitles.rvm.addic7ed/addic7ed/simple_requests.py

```diff
@@ -111,7 +111,11 @@ def text(self) -> str:
         :return: Response payload as decoded text
         """
         if self._text is None:
-            self._text = self.content.decode(self.encoding)
+            charset = self.headers.get_content_charset()
+            if charset and len(charset) > 0:
+                self._text = self.content.decode(charset.lower())
+            else:
+                self._text = self.content
```

Comment:
_text attribute must not be assigned binary content because it violates the API convention. I'd rather use a fallback encoding, e.g. UTF-8 with `replace` attribute to skip non-supported byte octets.

Reply:
Fixed in 60ef524